packages/core/src/providers/loops.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * <div style={{backgroundColor: "#24292f", display: "flex", justifyContent: "space-between", color: "#fff", padding: 16}}>
+ * <span>Built-in <b>Loops</b> integration.</span>
+ * <a href="https://loops.so">
+ *  <img style={{display: "block"}} src="https://authjs.dev/img/providers/loops.svg" width="48"/>
+ * </a>
+ * </div>
+ *
+ * @module providers/loops
+ */
+
+import type { EmailConfig } from "./email.js"
+import type { Awaitable, Theme } from "../types.js"
+
+export interface LoopsConfig extends EmailConfig {
+  apiKey?: string
+  transactionalId?: string
+  sendVerificationRequest: (params: {
+    identifier: string
+    url: string
+    expires: Date
+    provider: LoopsConfig
+    token: string
+    theme: Theme
+    request: Request
+  }) => Awaitable<void>
+  options: LoopsUserConfig
+}
+
+export type LoopsUserConfig = Omit<Partial<LoopsConfig>, "options" | "type">
+
+export default function Loops(config: LoopsUserConfig): LoopsConfig {
+  if (!config.transactionalId)
```

Comment:
If `transactionalId` is required, let's not make it optional in the types 👍

Reply:
updated 😄 
thanks for that, good catch